rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    person.rentals << self
+    @book = book
+    book.rentals << self
+  end
```

Comment:
When the wrong input is given, the app crashes specifically for creating Rentals. The app should not crash rather it should show a warning that the Wrong Input is given. Kindly fix this 👍🏻